docker-compose.yml

```diff
@@ -147,6 +147,42 @@ services:
       # <host_machine_directory>:<container_directory> – mirror bidirectional path in local context with path in Docker container
       - ./logs/dht-node:/logs/dht-node
 
+  ########################################################
+  # FEDERATION ###########################################
+  ########################################################
+  federation:
+    # name the image so that it cannot be found in a DockerHub repository, otherwise it will not be built locally from the 'dockerfile' but pulled from there
+    image: gradido/federation:local-production
+    build:
+      # since we have to include the entities from ./database we cannot define the context as ./federation
+      # this might blow build image size to the moon ?!
+      context: ./
+      dockerfile: ./federation/Dockerfile
+      target: production
+    networks:
+      - internal-net
+      - external-net
+    ports:
+      - 5000:5000
```

Comment:
Should this be 5001? I think on 5000 sits the DHT

Reply:
as discussed it is changed to 5010 for api-version 1_0